package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@brainhubeu/react-carousel",
-  "version": "1.3.0",
+  "version": "1.3.1",
```

Comment:
because it's the new feature I'd say we should increase minor version. What do you think?